pkg/apis/serving/k8s_validation.go

```diff
@@ -456,11 +456,17 @@ func validateReadinessProbe(p *corev1.Probe) *apis.FieldError {
 	// PeriodSeconds == 0 indicates Knative's special probe with aggressive retries
 	if p.PeriodSeconds == 0 {
 		if p.FailureThreshold != 0 {
-			errs = errs.Also(apis.ErrDisallowedFields("failureThreshold"))
+			errs = errs.Also(&apis.FieldError{
+				Message: "failureThreshold is disallowed when periodSeconds is zero",
+				Paths:   []string{"periodSeconds", "failureThreshold"},
```

Comment:
PeriodSeconds probably can be omitted from the path, since its value is valid.

Reply:
Thank you @vagababov Yes, updated.